gravitee-apim-portal-webui-next/src/app/api-details/api-tab-details/api-tab-details.component.ts

```diff
@@ -13,13 +13,49 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component } from '@angular/core';
+import { AsyncPipe, DatePipe } from '@angular/common';
+import { Component, Input, OnInit } from '@angular/core';
+import { of, switchMap } from 'rxjs';
+
+import { MarkdownPipe } from '../../../components/pipe/markdown.pipe';
+import { ApiService } from '../../../services/api.service';
+import { PageService } from '../../../services/page.service';
 
 @Component({
   selector: 'app-api-tab-details',
   standalone: true,
-  imports: [],
+  imports: [MarkdownPipe, AsyncPipe, DatePipe],
   templateUrl: './api-tab-details.component.html',
   styleUrl: './api-tab-details.component.scss',
 })
-export class ApiTabDetailsComponent {}
+export class ApiTabDetailsComponent implements OnInit {
+  @Input() apiId!: string;
+  @Input() apiVersion!: string;
+  @Input() apiLastUpdate!: Date | undefined;
+  @Input() apiCategories: Array<string> | undefined;
+  content!: string;
+
+  constructor(
+    private apiService: ApiService,
+    private pageService: PageService,
+  ) {}
+
+  ngOnInit(): void {
+    this.pageService
```

Comment:
Instead of a `.subscribe` to give content it's value, you can create a variable `content$: Observable<string>`, use `@if (content$ | async; as content)` and give that to the renderer. This will allow Angular to better control the unsubscribe.

Reply:
Done